### PR TITLE
Ant to Maven Migration

### PR DESCRIPTION
**Pull request by bparmar-splunk**
### Ant to Maven Migration
Below are the list of changes:

- Converted ant targets to its respective Maven builds
- Maven Parent - Child project structure implemented.
- As per Maven project structure, three modules (splunk, util & tests) are clubbed into one sub-module as splunk and examples is another sub-module.
- Maven deploy into Local Environment.
- All files used in Test cases are re-located from java/ directory to resources/ directory with regards to Maven resource handling.

### CODE REVIEW

File: splunk/pom.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M5</version>
+                <configuration>
+                    <skipTests>false</skipTests>
```
Comment (zenmoto) [inline splunk/pom.xml:65]: Could you change this value to `${skipTests}` to allow skipping tests from the command-line?

File: splunk/pom.xml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    <artifactId>splunk</artifactId>
+    <version>1.6.5</version>
+    <parent>
+        <artifactId>splunk-sdk-java</artifactId>
+        <groupId>com.splunk</groupId>
+        <version>1.0.0</version>
+    </parent>
+    <properties>
+        <!-- <jvm.arg.https.protocol>-Dhttps.protocols=TLSv1.1,TLSv1.2</jvm.arg.https.protocol> -->
```
Comment (zenmoto) [inline splunk/pom.xml:15]: It seems like we have some substantial work to do on the javadoc- could you add a property here to disable javadoc generation for the moment and add another bug to clean up javadoc generation?

`<maven.javadoc.skip>true</maven.javadoc.skip>`

**Review by zenmoto — COMMENTED** [review]
This looks really good.

File: splunk/pom.xml (hunk 2 above)
Comment (bparmar-splunk) [reply splunk/pom.xml:15]: Both changes are implemented and code is checked-in.

File: splunk/pom.xml (hunk 1 above)
Comment (bparmar-splunk) [reply splunk/pom.xml]: This change is implemented and code is checked-in.

**Review by zenmoto — APPROVED** [review]

**Pull request merged**